### PR TITLE
Stashing updates to review pr

### PR DESCRIPTION
**Pull request by JohnnyFoci**

### CODE REVIEW

File: Orbital-Docs/_pages/docs/designer_style_guide.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+permalink: /docs/designer_style_guide/
+sidebar:
+  nav: 'docs'
+---
+
+# Orbital Designer Style Guide
+
+Much of our style guide pulls directly from [Angulars Style Guide](https://angular.io/guide/styleguide). When in doubt , their documentation is a good place to look.
+
+##Guiding Principle: Consistency is kind.
```
Comment (mariareyesfoci) [inline Orbital-Docs/_pages/docs/designer_style_guide.md]: missing space after the double #

File: Orbital-Docs/_pages/docs/designer_style_guide.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+```javascript
+/**
+ * Completes event emitter
+ */
+```
+
+- Do not create formatted blocks of asterisks around comments. We find this to be a waste of space and time.
+
+## Language Guidelines
```
Comment (mariareyesfoci) [inline Orbital-Docs/_pages/docs/designer_style_guide.md]: remove subtitle

File: Orbital-Docs/_pages/docs/designer_style_guide.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Coding Style
```
Comment (mariareyesfoci) [inline Orbital-Docs/_pages/docs/designer_style_guide.md:57]: remove subtitle

File: Orbital-Docs/_pages/docs/designer_style_guide.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
+layout: single
+title: Orbital Designer Style Guide
+permalink: /docs/designer_style_guide/
+sidebar:
+  nav: 'docs'
+---
+
+# Orbital Designer Style Guide
```
Comment (mariareyesfoci) [inline Orbital-Docs/_pages/docs/designer_style_guide.md]: remove extra title

File: Orbital-Docs/_pages/docs/designer_style_guide.md (hunk 1 above)
Comment (JohnnyFoci) [reply Orbital-Docs/_pages/docs/designer_style_guide.md]: Awesome, also updated kind to king and added a newline underneath

**Review by mariareyesfoci — APPROVED** [review]

**Pull request merged**